src/components/NinDisplay.js

```diff
@@ -97,8 +119,6 @@ export class NinDisplay extends Component {
     );
```

Comment:
Perhaps it could improve readability to separate the redux hide/show logic from the render logic of any surrounding structures (i.e. streamlining the redux logic to not depend on position and only using position prop in return statement)? I might be wrong, but it looks like all this component needs to know about the nin is if any are saved so you can render for both cases (none: the text/link | any: number and button + separate logic handling what number looks like).